lib/document.js

```diff
@@ -1777,6 +1777,15 @@ Document.prototype.$__set = function(pathToMark, path, options, constructing, pa
  */
 
 Document.prototype.$__getValue = function(path) {
+  if (typeof path !== 'string' && !Array.isArray(path)) {
+    throw new TypeError(
+      'Invalid `path`. Must be either string or array. Got "' +
+      path +
+      '" (type ' +
+      typeof path +
+      ')'
```

Comment:
i think it would be a lot cleaner if template strings would be used here

Reply:
Fixed in 7b0dba0